submissions/Dovahkiin1991/classic-frogger-game/app.js

```diff
@@ -0,0 +1,124 @@
+//const
+const CELL_WIDTH = 101,
+    CELL_HEIGHT = 83,
+    ALL_ENEMIES = [50, 60, 30];
+const Enemy = function(y, index, player) {
```

Comment:
`index` is a generic term. Index of what? What is it in terms of the app's business domain?

Reply:
changed to enemyRowPosition